promdump/promdump.go

```diff
@@ -368,7 +376,118 @@ func exportMetric(ctx context.Context, promApi v1.API, metric string, beginTS ti
 			}
 		}
 	}
-	return writeFile(&values, filePrefix, fileNum)
+
+	return fileNum, writeFile(&values, filePrefix, fileNum)
+
+}
+
+// Holds custom metric file counts
+var customMetricCount uint
```

Comment:
I think I've changed my mind on this one. Let's put it up in the big `var` block at the top. There isn't really a good way to put this close to where it's used.

Reply:
I put `customMetricCount`  in the var block at the top